apps/emqx/i18n/emqx_schema_i18n.conf

```
@@ -569,6 +585,39 @@ emqx_schema {
     }
   }
 
+  persistent_session_builtin_session_table {
+    desc {
+         en: "Tuning for built-in session table."
+         zh: "对内置的会话表进行调整。"
+    }
+    label: {
+         en: "Persistent session."
```

If I remember correctly, the Label should not contain punctuation
```suggestion
         en: "Persistent session"
```